### PR TITLE
chore: run prettier on our yaml, json, markdown

### PR DESCRIPTION
**Pull request by ghost**
Normalize the formatting of these files too.

### CODE REVIEW

**Comment by JonasBa** [comment]
@bukzor-sentryio is this a one-off run or do we want to run this in CI (if so, mind checking how this impacts CI times)? I just wouldn't want to spend CI time formatting JSON fixtures 😅

**Review by asottile-sentry — CHANGES_REQUESTED** [review]
there's a few files in this that probably shouldn't be formatted -- like `static/app/data/world.json`

also the `+1.3M -31k` diff stat has me concerned that there's something github isn't showing us

**Comment by bukzor-sentryio** [comment]
> @bukzor-sentryio is this a one-off run or do we want to run this in CI (if so, mind checking how this impacts CI times)? I just wouldn't want to spend CI time formatting JSON fixtures 😅

It takes maybe minute for `run --all`, but runs in near-zero time for differential checks (the 99% path).

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,100 +1,111 @@
...
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: 'v3.0.3' # Use the sha or tag you want to point at
+    hooks:
+      - id: prettier
+        name: prettier (yaml, json, markdown)
+        types_or:
+          - json
+          - yaml
+          - markdown
+        exclude_types: [binary]
```
Comment (asottile-sentry) [inline .pre-commit-config.yaml:110]: this doesn't do anything because `binary` is not included in `types_or`

Comment (bukzor-sentryio) [reply .pre-commit-config.yaml]: Fixed, now.

**Review by asottile-sentry — APPROVED** [review]

**Pull request merged**